src/azure-cli/azure/cli/command_modules/network/_params.py

```diff
@@ -2146,6 +2146,7 @@ def load_arguments(self, _):
         c.argument('ssh_key', help='SSH key file location for SSH connections.', options_list=['--ssh-key'])
     with self.argument_context('network bastion rdp') as c:
         c.argument('disable_gateway', arg_type=get_three_state_flag(), help='Flag to disable access through RD gateway.')
+        c.argument('configure', help='Flag to configure RDP session.')
```

Comment:
Do you think it should be `get_three_state_flag` type as `disable_gateway`?

Reply:
@necusjz The flag here is just to trigger a different behavior - opening the Remote Desktop Connection UI instead of directly connecting to the host - compared to a value that is persisted.

This was my reasoning for not setting the `arg_type` based on my understanding on what the `get_three_state_flag` type means (explicit true or false field). Let me know if my understanding is incorrect here though.

Reply:
@PramodValavala-MSFT Here is the reference https://github.com/Azure/azure-cli/blob/dev/doc/authoring_command_modules/authoring_commands.md#registering-flags.

Reply:
@necusja Thanks for sharing that! This would more be a switch based on the docs, so I will update accordingly.